src/overlays/actors/ovl_En_Toryo/z_en_toryo.c

```diff
@@ -99,8 +100,6 @@ static EnToryoAnimation sEnToryoAnimation = { 0x06000E50, 1.0f, 0, 0 };
 
```

Comment:
And here. There are probably others in some of these files I didn't see in the preview.